demos/browser/app/meeting/CloudWatchLogger.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import LogLevel from '../../../../src/logger/LogLevel';
+import {ConsoleLogger} from '../../../../src/index';
+
+export default class CloudWatchLogger extends ConsoleLogger {
+  name: string;
+  level: LogLevel;
+  logCapture: string[] = [];
+  meetingID: string;
+  attendeeID: string;
+  logSeqNo: any;
+  lock = false;
+  batchSizes = 50;
```

Comment:
Please make some properties ```private```. Also, you can make this static constant like 
https://github.com/aws/amazon-chime-sdk-js/blob/a0cfc9611c848b69b2f1839dd91f867dc4d32934/demos/browser/app/meeting/meeting.ts#L33

```
private static BATCH_SIZE = 16;
```